pipeline/metadata/6.0.yaml

```diff
@@ -499,6 +499,16 @@ pipelines:
             ibmc: "conf/inventory/ibm-vpc-rhel-9-latest.yaml"
           metadata:
             - rbd
+        - name: "RGW bucket notifications testing on Multisite"
+          execution_time: "2h 8m 36s"
+          suite: "suites/pacific/rgw/tier-2_rgw_ms_test-bucket-notifications.yaml"
+          global-conf: "conf/pacific/rgw/rgw_multisite"
```

Comment:
IMO, this should be pointing to quincy not pacific

Reply:
@anrao19 , quincy suite contains multisite replication bucket notification tests which are fixed in 6.1
so executing pacific suite only for 6.0